package.json

```diff
@@ -1,30 +1,27 @@
 {
   "name": "viktorbot",
-  "version": "2.6.0",
-  "description": "Custom bot for the /r/viktormains Discord channel!",
+  "version": "2.0.0",
+  "description": "",
```

Comment:
Why removing the the description?
```suggestion
  "description": "Viktor Mains Discord Bot",
```

Reply:
I think it got changed involuntarily when I `npm init`ed after removing everything from previous repo